src/utils/fetchAttestations/createAttestationFetcher.ts

```diff
@@ -66,12 +73,15 @@ export function createAttestationFetcher(round: Round): AttestationFetcher {
           ...filter,
           where: {
             schemaId: { in: schemas },
-            revoked: { equals: false },
+            ...(includeRevoked ? {} : { revoked: { equals: false } }),
             time: { gte: startsAt },
             ...filter?.where,
           },
         },
       }),
-    }).then((r) => r.data?.attestations.map(parseAttestation));
+    }).then((response) => {
+      noCache && void response.ejectCache();
```

Comment:
This creates a long trail for parameters to pass between functions. I would suggest doing some change in the variables sent to the fetch to trigger a new refetch. 

This could be a nonce or timestamp value for one query param we're not currently using:
https://studio.apollographql.com/sandbox/explorer

`expirationTime` could be a good candidate.